osbuild/buildroot.py

```diff
@@ -257,9 +260,40 @@ def run(self, argv, monitor, stage_timeout=None, binds=None, readonly_binds=None
             api_path = "/run/osbuild/api/" + api.endpoint
             mounts += ["--bind", api.socket_address, api_path]
 
+        index = osbuild.meta.Index(self._libdir)
```

Comment:
The idea here is for a given manifest, get all the stages and then get the schema for the given stage. 

After I have these details, a new field can be added to include what capabilities are needed. 

Then the correct caps can be dropped from bwrap

Reply:
This is a replication of the data and code found in main_cli.py , perhaps this data can be passed once from main_cli.py instead of being generated here again.

Something to look at when we get to optimisation